website/src/main/java/com/google/step/servlets/DataServlet.java

```diff
@@ -14,6 +14,11 @@
 
 package com.google.step.servlets;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Entity;
```

Comment:
optional: I usually organize my imports in alphabetical order, so Entity would come before Key